src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -536,10 +540,10 @@ public function mirror(string $originDir, string $targetDir, \Traversable $itera
             }
         }
 
-        $copyOnWindows = $options['copy_on_windows'] ?? false;
+        $followSymlinks = isset($options['follow_symlinks']) && $options['follow_symlinks'] || isset($options['copy_on_windows']) && $options['copy_on_windows'];
```

Comment:
This is working fine, but I'm personally prefer group condition options with parentheses for readability
E.g. 
`(isset($options['follow_symlinks']) && $options['follow_symlinks']) || (isset($options['copy_on_windows']) && $options['copy_on_windows']);`

But this is not mandatory to change